install/install.sh

```diff
@@ -288,6 +273,85 @@ check_bin_dir() {
   fi
 }
 
+print_install() {
+  # if the shell does not fit the default case change the config file
+  # and or the config cmd variable
+  for s in "bash" "zsh" "ion" "tcsh" "xonsh" "fish"
+  do
+    case ${s} in
+      ion )
+        # shellcheck disable=SC2088
+        # we don't want these '~' expanding
+        config_file="~/.config/ion/initrc"
+        config_cmd="eval \$(starship init ${s})"
+        ;;
+      fish )
+        # shellcheck disable=SC2088
+        config_file="~/.config/fish/config.fish"
+        config_cmd="starship init fish | source"
+        ;;
+      tcsh )
+        config_cmd="eval \`starship init ${s}\`"
+        ;;
+      xonsh )
+        config_cmd="execx(\$(starship init xonsh))"
+        ;;
+      * )
+        # shellcheck disable=SC2088
+        config_file="~/.${s}rc"
+        config_cmd="eval \"\$(starship init ${s})\""
+        ;;
+    esac
+
+    printf "  %s\n  Add the following to the end of %s:\n\n\t%s\n\n" \
+      "${BOLD}${UNDERLINE}${s}${NO_COLOR}" \
+      "${BOLD}${config_file}${NO_COLOR}" \
+      "${config_cmd}"
+  done
```

Comment:
This doesn't work correctly, the `config_file` isn't reset to the default so the `tcsh` and `xonsh`  instructions both have the wrong config file in the instructions.

```
> Please follow the steps for your shell to complete the installation:
...
 
  ion
  Add the following to the end of ~/.config/ion/initrc:

        eval $(starship init ion)

  tcsh
  Add the following to the end of ~/.config/ion/initrc:

        eval `starship init tcsh`

  xonsh
  Add the following to the end of ~/.config/ion/initrc:

        execx($(starship init xonsh))

 ...
```


```suggestion
  for s in "bash" "zsh" "ion" "tcsh" "xonsh" "fish"
  do
    # shellcheck disable=SC2088
    # we don't want these '~' expanding
    config_file="~/.${s}rc"
    config_cmd="eval \"\$(starship init ${s})\""
 
    case ${s} in
      ion )
        # shellcheck disable=SC2088
        config_file="~/.config/ion/initrc"
        config_cmd="eval \$(starship init ${s})"
        ;;
      fish )
        # shellcheck disable=SC2088
        config_file="~/.config/fish/config.fish"
        config_cmd="starship init fish | source"
        ;;
      tcsh )
        config_cmd="eval \`starship init ${s}\`"
        ;;
      xonsh )
        config_cmd="execx(\$(starship init xonsh))"
        ;;
    esac

    printf "  %s\n  Add the following to the end of %s:\n\n\t%s\n\n" \
      "${BOLD}${UNDERLINE}${s}${NO_COLOR}" \
      "${BOLD}${config_file}${NO_COLOR}" \
      "${config_cmd}"
  done
```

Reply:
Thanks for the double check.